WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java

```diff
@@ -436,6 +437,8 @@ private void renderPost(int position, ReaderPostViewHolder holder) {
             params.topMargin = titleMargin;
         }
 
+        holder.visit.setContentDescription(holder.cardView.getContext().getString(R.string.reader_label_visit) + " " + post.getTitle());
```

Comment:
I'm not sure about this. First, is it locale-friendly to append the post title to the word "Visit?" Also, it seems a bit confusing to have "Visit [Title of Post]" - it's not clear to me that I'll be visiting the post unless I already know that's the title of the post. How about just updating the layout XML instead and using "Visit post" or the existing "View in browser" (`R.string.view_in_browser`)?